.github/workflows/CI.yaml

```diff
@@ -46,34 +46,33 @@ jobs:
       run: |
         sudo chown -R $(id -u):$(id -g) ${CONDA}
 
-    - name: Create environment for package
-      shell: bash
-      run: |
-        . devtools/github-actions/initialize_conda.sh
-        conda activate
-        conda info
-        python devtools/scripts/create_conda_env.py -n=test -p=$PYVER devtools/conda-envs/test_env.yaml
+    - name: Setup Conda
+      uses: conda-incubator/setup-miniconda@v1
```

Comment:
The OpenFF folks were just talking about `v2` update here: https://github.com/openforcefield/status/issues/13

Reply:
I bumped that in #411 since I missed it in this one.